src/tvuikit.cs

```diff
@@ -290,4 +300,166 @@ interface TVCollectionViewFullScreenLayout {
 		[Export ("transitioningToCenterIndexPath")]
 		bool TransitioningToCenterIndexPath { [Bind ("isTransitioningToCenterIndexPath")] get; }
 	}
+
+	[TV (15,0), NoWatch, NoMac, NoiOS]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface TVMediaItemContentConfiguration : UIContentConfiguration, NSSecureCoding {
+
+		[Static]
+		[Export ("wideCellConfiguration")]
+		TVMediaItemContentConfiguration CreateWideCell ();
```

Comment:
Mixed feelings about adding Configuration at the end.

Reply:
Yeah I thought so but the return type is a configuration object and to add to it you have to type the whole type name because it is a static method

Reply:
`Configuration` suffix or `CreateForWideCell` ?
it's definitively not a wide cell that comes out :)